backend/src/main/java/org/example/backend/controller/UserController.java

```diff
@@ -4,49 +4,57 @@
 import org.example.backend.model.User;
 import org.example.backend.service.UserService;
 import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
 import org.springframework.security.core.annotation.AuthenticationPrincipal;
 import org.springframework.security.oauth2.core.user.OAuth2User;
 import org.springframework.web.bind.annotation.*;
 import org.springframework.web.server.ResponseStatusException;
 
+import java.util.List;
+
 @RestController
 @RequestMapping("/api/users")
 @RequiredArgsConstructor
 public class UserController {
 
-    private final UserService service;
+    private final UserService userService;
 
+    @GetMapping
+    public ResponseEntity<List<User>> getAllUsers() {
+        List<User> users = userService.getAllUsers();
+        return new ResponseEntity<>(users, HttpStatus.OK);
```

Comment:
Why do you use "ResponseEntity" with this? Is it an oauth2 thing?

Reply:
This is actually an artefact from previous versions of my UserControllerTest. I don't need it here, anymore. Thank you a lot for your keen eye and pointing it out :-)